lib/slack_notifier.rb

```diff
@@ -0,0 +1,10 @@
+class SlackNotifier
+  def initialize(channel)
+    @notifier = Slack::Notifier.new(ENV['SLACK_NOTIFIER_WEBHOOK_URL'], channel: '#jobs', username: 'OpenJobs')
```

Comment:
where are u using `channel` ? :)
Here I liked how you passed the channel in the constructor and pass the message at notifiy parameter :)